words/all.asm

```diff
@@ -38,27 +54,15 @@ xt_cold:
                 sta output+1
 
                 ; Load all of the important zero page variables from ROM
-                ldx #cold_zp_table_end-cold_zp_table-1
+                ldy #cold_zp_table_end-cold_zp_table-1
 
 _load_zp_loop:
                 ; This loop loads them back to front. We can use X here
                 ; because Tali hasn't started using the stack yet.
-                lda cold_zp_table,x
-                sta zpage,x
-                dex
-                bne _load_zp_loop
-
-                ; Copy the 0th element.
```

Comment:
switched this loop to y so x can be set for data stack earlier, and avoid this extra 0th step with bpl